Sources/ClientRuntime/Networking/Http/CRT/CRTClientEngine.swift

```diff
@@ -42,9 +42,9 @@ public class CRTClientEngine: HttpClientEngine {
             )
 
             var socketOptions = SocketOptions(socketType: .stream)
-    #if os(iOS) || os(watchOS)
+#if os(iOS) || os(watchOS)
```

Comment:
does `tvOS` matter here?

Reply:
I just reformat it. tvOS story is same as we have other places.